docpages/example_code/oggopus.cpp

```diff
@@ -37,60 +37,122 @@ int main() {
 			/* Tell the user we joined their channel. */
 			event.reply("Joined your channel!");
 		} else if (event.command.get_command_name() == "play") {
+
 			/* Get the voice channel the bot is in, in this current guild. */
-			dpp::voiceconn* v = event.from->get_voice(event.command.guild_id);
+			dpp::voiceconn* v = event.from->get_voice(event.channel.guild_id);
```

Comment:
this has been changed to something that won't compile - slash command event doesn't have a channel member

Reply:
I flipped the two files around as oggopus was originally the second example :P